website/src/categories/components/tooltip/usage.md

```diff
@@ -5,27 +5,82 @@ eleventyNavigation:
   parent: Tooltip
   key: TooltipUsage
 ---
+<section class="no-heading">
+<div class="ds-example">
+
+<sl-button aria-describedby="tooltip" fill="solid" variant="primary">Button with a tooltip</sl-button>
+<sl-tooltip id="tooltip">This is a tooltip with some additional information</sl-tooltip>
+
+</div>
+
+<div class="ds-code">
+
+  ```html
+<sl-button aria-describedby="tooltip" fill="solid" variant="primary">Button with a tooltip</sl-button>
```

Comment:
Again, probably best to only focus on the tooltip. So remove `fill` and `variant`.